eng/Version-Increment.ps1

```diff
@@ -17,85 +17,128 @@ The Name of the Package
 .PARAMETER PackageDirName
 Used in the case where the package directory name is different from the package name. e.g in cognitiveservice packages
 
-.EXAMPLE
-Updating package version for core
+.PARAMETER NewVersionString
+Use this to overide version incement logic and set a version specified by this parameter
+
 
+.EXAMPLE
```

Comment:
Can you please add some docs about this script in Versioning.md file?

Reply:
Done